e2e/tests/core/03-connection/connection_test.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	paramsproposaltypes "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
 	"github.com/strangelove-ventures/ibctest/v6/ibc"
-	"github.com/strangelove-ventures/ibctest/v6/test"
+	test "github.com/strangelove-ventures/ibctest/v6/testutil"
```

Comment:
In the future it might be nice to use a more descriptive alias. "test" is very generic